app/services/idv/flows/doc_auth_flow.rb

```diff
@@ -23,9 +23,9 @@ class DocAuthFlow < Flow::BaseFlow
         { name: :getting_started },
         { name: :verify_id },
         { name: :verify_info },
-        *([name: :secure_account] if !IdentityConfig.store.gpo_personal_key_after_otp),
+        { name: :secure_account },
         { name: :get_a_letter },
-        *([name: :secure_account] if IdentityConfig.store.gpo_personal_key_after_otp),
+        { name: :secure_account },
```

Comment:
I'm expecting there'd only be one of the `:secure_account`s left after removing the feature flag, presumably the latter?

Reply:
Good catch; thank you.

Reply:
I looked into why the tests didn't smack me for this, and the tests use the same constants as the code; seems like that weakens the test net. I think we should be checking for explicit values in the test, instead of checking that we use the correct constants. As it stands, the tests don't verify that the constants are correct. Changing them.

Reply:
Made the tests actually _complain_ about the extra step, and then removed it.